client_unix_test.go

```diff
@@ -0,0 +1,44 @@
+// +build darwin dragonfly freebsd linux nacl netbsd openbsd solaris
```

Comment:
I'd suggest enumerating platforms without unix socket support here, since this list is shorter.